crates/core_arch/src/powerpc/mod.rs

```diff
@@ -1,4 +1,8 @@
 //! PowerPC intrinsics
+
+#[macro_use]
+pub(crate) mod macros;
```

Comment:
This doesn't need to be `pub(crate)`, it's only used in the `powerpc` module.

Reply:
Sure :)